packages/flutter_tools/lib/src/codegen.dart

```diff
@@ -178,6 +178,7 @@ class CodeGeneratingResidentCompiler implements ResidentCompiler {
     List<Uri> invalidatedFiles, {
       String outputPath,
       PackageConfig packageConfig,
+      bool suppressErrors = false,
```

Comment:
Can you add a comment for when you would want to suppress errors?  Doesn't sound like something you would generally want to do.

Reply:
done here: https://github.com/flutter/flutter/pull/58539/files#diff-e78651ad5bd5c46811f2b76bc465220bR375

Reply:
I guess this PR is the crumb trail for the signature as to why you would want to suppress errors (so they aren't displayed twice in certain scenarios).